src/App.scss

```diff
@@ -1,15 +1,3 @@
-.App {
-  display: flex;
-  height: 100vh;
-}
-
-.App-header {
-  width: 30vw;
-  background-color: #282c34;
-  color: white;
-}
-
 .App-main {
-  flex: 1;
-  overflow-y: auto;
+  padding-top: 176px;
```

Comment:
This seems quite magical and probably won't work if font sizes are different. As far as I can tell these material components from `rmwc` are all based on `rem`.

However, I don't really know what a proper solution would be. We can use [`<TopAppBarFixedAdjust />`](https://github.com/jamesmfriedman/rmwc/tree/master/src/top-app-bar#topappbarfixedadjust) to fill in the gap for the app bar itself, but we cannot easily reproduce the height of the title text + icon.

Or we can just make everything static and make the `.App-main` scrollable, instead of making the full document scrollable. Thoughts?

Reply:
Pulled in the heights from the underlying mdc-lib, heights from prominent-app-bar and tabs.

I think this is still better than making App-main scrollable as we would lose a nice animation where a drop-shadow is conditionally rendered on the mdc/top-app-bar

Reply:
Neat!